code/game/jobs/job/security.dm

```diff
@@ -28,6 +28,7 @@
 	access = list(
 		ACCESS_COM_COMMS,
 		ACCESS_SEC_COMMS,
+		ACCESS_SECURITY, //this might break something, but if you see this, it probably didnt!
```

Comment:
did you need to add the comment for that

Reply:
yes